Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/settings/viewholder/SingleChoiceViewHolder.java

```diff
@@ -35,9 +35,169 @@ public void bind(SettingsItem item)
 
 		mTextSettingName.setText(item.getNameId());
 
-		if (item.getDescriptionId() > 0)
+		switch (item.getDescriptionId())
 		{
-			mTextSettingDescription.setText(item.getDescriptionId());
+			case R.string.dynamic_cpu_core:
```

Comment:
I am terribly sorry that I kept this one for too long to come back with this review. 

I don't think this should land in master as is, the way the SingleChoiceViewHolder was tightly coupled to the internal values of the SettingsItem is not scalable and prone to more issues. I think the better way to go is to change the SingleChoiceSetting from accepting one descriptionId to an array of descriptions kinda similar to how we pass the entries/values right now. So we can make it agnostic to which values you are using and avoid writing this long switch that is not gonna be scalable at all.